src/Explorer/ExplorerTable.jsx

```diff
@@ -55,7 +56,9 @@ export class ExplorerTableComponent extends Component {
         <TableData c_width={columnWidths[0]}>
           <Link to={`/${file.project_id}`}>{file.project_id}</Link>
         </TableData>
-        <TableData c_width={columnWidths[1]}>{file.name}</TableData>
+        <TableData c_width={columnWidths[1]}>
```

Comment:
we should not show them as link if user doesn't have read-storage permission on this project?